harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -671,8 +667,10 @@ def _load(self, load_path: pathlib.Path) -> None:
 
         if "model_state_dict" in checkpoint:
             # Backward compatible with older checkpoint format.
-            check.not_in("models_state_dict", checkpoint)
-            check.eq(len(self.context.models), 1)
+            if "models_state_dict" in checkpoint:
+                raise RuntimeError("Both model_state_dict and models_state_dict in checkpoint.")
+            if len(self.context.models) != 1:
+                raise RuntimeError()
```

Comment:
These checkpoint loading exceptions are technically loading user-provided values, and so need a real message.

"Old-format checkpoint cannot be loaded into a model with more than one checkpoint" would be fine.

Reply:
Thanks. I'm going to change the actual check to be `... > 1` to match the message.

Reply:
Waitasecond, do you mean "Old-format checkpoint cannot be loaded into a **context** with more than one **model**."

Reply:
Yeah, and I realized my typo after clicking "comment" but I was not concerned about you missing my typo.

You don't miss any of my typos lol.

Reply:
Done!